### DISCUSSION TITLE
Parts Inventory/Plates/Dryers

### DISCUSSION THREAD

**Discussion by jon-merchant** [MarksMakerSpace/filament-profiles#42] [Ideas]
What about having parts also available to add to "my parts"? This would also be pretty helpful - not sure how you'd want to do plates, but that would be nice too, have inventory of plates. I notice there's a dryer list, but no way to add to me, I know it's not a big deal and most people probably have 1 dryer, but that would be handy too.

**Comment by GE-Axiom** [comment]
I like this idea honestly considering 3dFilamentprofiles to be a one stop inventory system would be so rad.

**Reply by markcarroll** [reply]
I can look into adding other parts and making them user-added like filaments.

**Comment by markcarroll** [comment]
@jon-merchant good idea about adding parts to the "My" section.  I will add that to the list too.

**Reply by jon-merchant** [reply]
Thank you!

**Reply by markcarroll** [reply]
I haven't forgotten this and am planning it now.
